src/Workspaces/Core/MSBuild.BuildHost/BuildHost.cs

```diff
@@ -101,11 +101,7 @@ private bool TryEnsureMSBuildLoaded(string projectOrSolutionFilePath)
         }
     }
 
-#if NET472 || NET6_0 // If we're compiling against net472 or net6.0, we get our MemberNotNull from the workspaces assembly. It has it in the net6.0 case since we're consuming the netstandard2.0 version of Workspaces.
-    [workspaces::System.Diagnostics.CodeAnalysis.MemberNotNull(nameof(_buildManager))]
-#else // If we're compiling against net7.0 or higher, then we're getting it staright from the framework.
-    [System.Diagnostics.CodeAnalysis.MemberNotNull(nameof(_buildManager))]
-#endif
+    [MemberNotNull(nameof(_buildManager))]
```

Comment:
Where is this coming from in the net472/net60 scenario now?

Reply:
Previously we had to resolve ambiguity between workspaces and BCL. The conditional declarations are linked directly to the host now, so there is no ambiguity.